src/__tests__/value-object-parser-test.ts

```diff
@@ -69,7 +69,7 @@ describe('ValueObjectParser', function() {
     it('parses a value object with two properties and lots of custom ' +
        'information that is valid', function() {
       const valueFileContents = '%library name=RMSomethingLibrary\n' +
-                              '%type name=Foo library=Bar file=Baz canForwardDeclare=false\n' +
+                              '%type name=Foo library=Bar file=NSObject+Baz canForwardDeclare=false\n'
```

Comment:
You can't remove the plus at the end of the line, it's building a multiline string